src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -990,7 +992,7 @@ void goto_instrument_parse_optionst::instrument_goto_program()
   // now do full inlining, if requested
   if(cmdline.isset("inline"))
   {
-    do_indirect_call_and_rtti_removal();
+    do_indirect_call_and_rtti_removal(/*force=*/true);
```

Comment:
It might be better change the API to use an enum to make clear what the value means?

Reply:
I think having boolean flags for options should be fine.

Reply:
It surely is, but the fact that you felt compelled to add a comment suggests that an enum (with just two values!) could be helpful for people reading the code.

Reply:
I agree that it would be more readable. I'm slightly concerned about consistency with the rest of the code base though as having two-value enums for boolean flags isn't done commonly there. I believe I've added the comment explaining the argument here just randomly (I think it's also the case for lots of other function calls with boolean literals as arguments that it's not clear what the flag does unless one inspects the declaration/definition). I'll remove the comment from the call.

Reply:
I know that #1547 introduced one of those, and there was some PR (not sure whether that one or another) that got me very confused initially. Indeed it's not yet common practice, but likely we should look into adopting it. Because I'm in the mood: #1854.